Report safetensors version in transformers-cli env

sgugger opened the pull request:
# What does this PR do?

This PR adds `safetensors` to the info reported by `transformers-cli env` and in particular puts a note when safetensors is ignored becuase of a too old PyTorch (see #22370 )

sgugger committed:
Report safetensors version in transformers-cli env

HuggingFaceDocBuilderDev commented:
The docs for this PR live [here](https://moon-ci-docs.huggingface.co/docs/transformers/pr_22374). All of your documentation changes will be reflected on that endpoint.

amyeroberts reviewed (approved):
Thanks for adding!

gante committed:
TensorFlow: additional missing `cmake` dependencies in CI (#22383)

* missing cmake

* more cmake

sgugger committed:
Styling

Charlie-Bell committed:
Changed world_size() to get_world_size() bugfix (#22381)

Edited one line in src/transormers/generation/utils.py. Changed dist.world_size() to dist.get_world_size() since world_size() doesn't exist in pytorch.dist.

nickprock committed:
Translated documentation in italian (#22388)

* updated toctree

* added and translated mdx documents

sgugger committed:
Adapt find_tied_parameters to handle breaking change in Accelerate (#22360)

kooshi committed:
load_in_8bit now respects 'balanced' device maps in multi-gpu environments (#22377)

balanced 8bit memory

vsokolovskii committed:
Wav2Vec2ProcessorWithLM can return N best hypotheses now (#22235)

* Wav2Vec2ProcessorWithLM can return N best hypotheses now

Signed-off-by: Vladislav Sokolovskii <vladislav@parrothq.com>

* Wav2Vec2ProcessorWithLM n_best cannot be None

Co-authored-by: amyeroberts <22614925+amyeroberts@users.noreply.github.com>

* Batch decoding can return  N best hypotheses now

batch_decode was extended with the same functionality as decode
function, N best hypotheses per sample can be returned

Signed-off-by: Vladislav Sokolovskii <vladislav@parrothq.com>

---------

Signed-off-by: Vladislav Sokolovskii <vladislav@parrothq.com>
Co-authored-by: Vladislav Sokolovskii <vladislav@parrothq.com>
Co-authored-by: amyeroberts <22614925+amyeroberts@users.noreply.github.com>

sgugger committed:
Trigger CI maybe
